src/service/tempMailApi.js

```diff
@@ -6,11 +6,24 @@ class TemporalMailService {
   constructor() {
     this.BASE_API_URL = 'https://api.mail.tm';
     this.httpClient = require('axios');
+    this.emailDomain = '';
   }
+
+  async _getDomains() {
```

Comment:
Function without a return value??
well, there are allways new ways of programmin, not here please.